proposed-field-type-changes.txt

```diff
@@ -0,0 +1,141 @@
+* Check test data and keep data there consistent with proper type.
+* We are inconsistent with regards to lowerCamelCase  versus CapitalizedCamelCase.
+
+
+all
+  id: regex for DID
+    "type": "string",
+    "pattern": "^did:polygon:(mumbai|main):[0-9a-zA-Z]+$"
+  journeyId
+    "type": "string",
+    "format: "uuid"
+
+idscanPassport, idscanSelfie, proofOfResidence
+    documentType: "image/jpeg" | "image/png" |??
+    base64: {
+            "type": "string",
+            "media": {
+                "binaryEncoding": "base64",
```

Comment:
i am not sure if polygon-id supports this binaryEncoding feature, let's try it would be awesome if it works